### ISSUE TITLE
Commit convention consistency for major version bumps 

### ISSUE THREAD

**Issue by moaazsidat** [semantic-release/semantic-release#1720]
## New feature motivation & description
👋  Hiya! Big fan of semantic-release & thankful to the authors/contributors for building/maintaining the tool!

One thing that irks me every time I've used semantic-release is how the major version bump requires following a commit convention that deviates from the others. 

For patch: `fix: ...`
For minor/feature: `feat: | feature: ...`
For major `BREAKING CHANGE: ...`, **_but it should show up in the footer of the commit message_**. 

Wouldn't it be a much better developer experience to make major follow the same commit convention as patch and minor. 

I understand that some of this comes from the [Angular Commit Convention](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) as previously noted, but it's just that one little behaviour that behaviours slightly differently causing some amount of time debugging it when first using the tool. I also realize that this is configurable, however, perhaps my inclination is that the tool could support ideal defaults. 

Wondering if the authors/contributors have an opinion about this, happy to spin up a PR updating this behaviour if this is something that'd be worthwhile, thanks!

**Comment by gr2m** [comment]
Personally, I agree, I'm not a fan of the current default commit convention either. But it is configurable, and you can even define your own. See https://github.com/semantic-release/commit-analyzer. Does that work for you?

**Comment by moaazsidat** [comment]
Yep, as noted above, the configuration definitely offers the flexibility, but was more so interested in changing it so that the [defaults are sensible](https://stevebennett.co/2017/07/24/the-power-of-sensible-defaults/) (better developer experience) for first time users and such.
